model/src/test/kotlin/VcsInfoTest.kt

```diff
@@ -100,7 +99,7 @@ class VcsInfoTest : WordSpec({
 
             with(exception) {
                 propertyName shouldBe "unknown"
-                knownPropertyIds should containAll<Any>("type", "url", "revision", "resolved_revision", "path")
+                knownPropertyIds should containAll<Any>("type", "url", "revision", "path")
```

Comment:
Any idea why the test still passed? Again a Gradle task caching issue?

Reply:
Ah, scratch that, I had overlooked that `resolved_revision` was still present a few lines above in the test data.